src/dev/impl/DevToys/ViewModels/Tools/Converters/NumberBaseConverter/BasicNumberBaseConverterControlViewModel.cs

```diff
@@ -0,0 +1,291 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DevToys.Api.Core;
+using DevToys.Api.Core.Settings;
+using DevToys.Core.Threading;
+using DevToys.Shared.Core.Threading;
+using DevToys.Api.Tools;
+using DevToys.Models;
+using DevToys.UI.Controls;
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.Toolkit.Mvvm.Messaging;
+using Windows.UI.Xaml;
+using DevToys.Core;
+using DevToys.Messages;
+
+namespace DevToys.ViewModels.Tools.Converters.NumberBaseConverter
+{
+    [Export(typeof(BasicNumberBaseConverterControlViewModel))]
```

Comment:
No need to do an Export since you instantiate it manually in the dependency property in .xaml.cs

Reply:
It is there any way to prevent manual instantiation for both view models?

Reply:
Yes, alternatively, you could import this view model into the main one (NumberBaseConverterToolViewModel) with something like:

```
[ImportingConstructor]
public NumberBaseConverterToolViewModel(BasicNumberBaseConverterControlViewModel viewmodel)
{
}
```